brainsets/datasets/__init__.py

```
@@ -18,11 +18,19 @@
     "KempSleepEDF2013",
 ]
 
+_openneuro_datasets = [
+    "OpenNeuroDataset",
+    "KlinzingSleepDS0055552024",
+    "KochiVisualNamingDS0069142025",
+    "ShiraziHBNR1DS0055052024",
+]
+
 __all__ = (
```

Ruff (0.15.10)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)